apps/galaxy/src/__tests__/modules/auth/SignUpForm/SignUpForm.test.tsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import renderWithMockedProvider from "@src/__tests__/testHelper";
 import userEvent from "@testing-library/user-event";
 import { screen } from "@testing-library/react";
+import MockAdapter from "axios-mock-adapter";
```

Comment:
@VanderDeng Did we are settled on using `axios-mock-adapter` instead of `msw` to mock our api requests? If so, I might refactor mine test suites to make sure our tests is consistent.

Reply:
yep, try to use `msw` instead of `axios-mock-adapter`

Reply:
Okay, as per convo, we will migrate to `msw` in a separate ticket